### PR TITLE
Added kabyle layout

### PR DESCRIPTION
**Pull request by BoFFire**
Hi,

I added Kabyle layout for Futo keyboard project based on my work on Heliboard.

Regards,

### CODE REVIEW

**Comment by abb128** [comment]
Can you add kab to mapping.yaml?

**Comment by BoFFire** [comment]
> Can you add kab to mapping.yaml?

@abb128 Added kab to mapping. I hope it is okay.

**Comment by tenextractor** [comment]
Wouldn't it be better to add all letters instead of hiding them behind long presses? With a layout like this:
```
č ḍ ɛ ǧ ɣ ḥ ṣ ṭ ẓ
a z e r t y u i o p
q s d f g h j k l m
w x c v b n
```
This would give easy access to the letters Ẓ, Ṛ, Ṭ, O, Ṣ, Ḍ, Ǧ, Ḥ, Č, P instead of having to hold down keys for them.
It also makes typing URLs or any ASCII data easier, without having to switch back to an English layout.

**Comment by BoFFire** [comment]
> Wouldn't it be better to add all letters instead of hiding them behind long presses? With a layout like this:
> 
> ```
> č ḍ ɛ ǧ ɣ ḥ ṣ ṭ ẓ
> a z e r t y u i o p
> q s d f g h j k l m
> w x c v b n
> ```
> 
> This would give easy access to the letters Ẓ, Ṛ, Ṭ, O, Ṣ, Ḍ, Ǧ, Ḥ, Č, P instead of having to hold down keys for them. It also makes typing URLs or any ASCII data easier, without having to switch back to an English layout.

Thank you very much for your suggestion. Yes it could be a good variant.

I'm testing your suggestion but I have to ask our community.
IMG_20241107_181303

File: mapping.yaml [hunk 1]
```diff
@@ -365,6 +365,18 @@ languages:
...
+    - swiss
+    - qwerty
+    - qwertz
+    - dvorak
+    - colemak
+    - colemak_dh
+    - colemak_dh_ansi
+    - bepo
+    - pcqwerty
+    - nordic
```
Comment (abb128) [inline mapping.yaml:379]: You need to add `kabyle` to this list, probably at the top, else the `kabyle.yaml` layout will not be presented as an option.

Comment (BoFFire) [reply mapping.yaml]: @abb128  Added kabyle to mapping.yaml

**Pull request merged**